src/libs/tagging.c

```diff
@@ -624,6 +1097,20 @@ void gui_init(dt_lib_module_t *self)
   gtk_box_pack_start(hbox, button, FALSE, TRUE, 0);
   g_signal_connect(G_OBJECT(button), "clicked", G_CALLBACK(export_button_clicked), (gpointer)self);
 
+  button = gtk_button_new_with_label(_("tree"));
+  d->toggle_tree_button = button;
+  gtk_widget_set_tooltip_text(button, _("toggle list / tree view"));
+  dt_gui_add_help_link(button, "tagging.html#tagging_usage");
+  gtk_box_pack_end(hbox, button, FALSE, TRUE, 0);
+  g_signal_connect(G_OBJECT(button), "clicked", G_CALLBACK(toggle_tree_button_callback), (gpointer)self);
+
+  button = gtk_button_new_with_label(_("sugg"));
```

Comment:
The is too many buttons on the same line. This goes out of screen with French translation at least. We have:

new / delete / import / export / rename / tree / sugg

I would propose, 2 lines:

import / export / suggestion
new / delete / rename

And I would put the toggle for the tree/iist with a small icon on the right of suggestion.

Reply:
I've split in two lines as suggested. The other option would have been to remove the buttons delete & rename to keep only the click-right sub-menu.
I'm ok to use icons but I need to understand how I've to do that. I that case I think both suggestion/no suggestion and tree/list should be icons. For suggestion/no suggestion + / - could be good, couldn't it ?

Reply:
done